packages/backend/package.json

```diff
@@ -9,6 +10,7 @@
     "bcryptjs": "^2.4.3",
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
+    "crypto": "^1.0.1",
```

Comment:
Is crypto not a core node module now? I belive the npm package itself is no longer supported, and we should just use the one available in node itself.